helm/ffc-demo-payment-service/templates/service.yaml

```
@@ -3,9 +3,7 @@ kind: Service
 apiVersion: v1
 metadata:
   name: {{ .Values.postgresHost }}
-  labels:
-    environment: {{ quote .Values.environment }}
-  namespace: {{ .Values.namespace }}
+  labels: {{ include "ffc-demo.labels" . | trimSuffix "\n" | indent 2 }}
```

Accidental removal of namespace
```suggestion
  labels: {{ include "ffc-demo.labels" . | trimSuffix "\n" | indent 2 }}
  namespace: {{ .Values.namespace}}
```